### PR TITLE
Merge 18/12/03

### PR DESCRIPTION
**Pull request by yeskunall**
I had to push some of my heavy changes (mainly app structure) in order to restore the app back to how it was during the demo. Please read the `README` so you know what to do in order to setup Talos’ client successfully.

Pull this branch. Test it out. Ping me if you have any questions. 

@abungay I think you should be good to go now, right? LMK if you need me for anything else. If everything looks good, approve this PR, and I’ll merge it into `master`.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Getting started
+
+- Pull the repository
+- Sign up for a [ZenHub](https://www.zenhub.com/) account
+
+## Setup
+
+`cd` into the project
+
+Copy the contents of `env.example` into `.env` like so:
```
Comment (abungay) [inline CONTRIBUTING.md]: what's this for?

Comment (yeskunall) [reply CONTRIBUTING.md]: Here, just read [this](https://medium.com/@ktruong008/absolute-imports-with-create-react-app-4338fbca7e3d).

Comment (abungay) [reply CONTRIBUTING.md]: cool, I like that but I've read before that you shouldn't commit the .env file so is this a hacky way to bypass that?

**Comment by yeskunall** [comment]
No, we're not pushing the file. Notice it's nowhere in our source control.
Precisely why I tell the dev to copy the example env file into the actual
file. That file is in our list of ignored files. Does that explain it? If
so, please resolve this conversation. I can't because I'm repmy via email.

File: CONTRIBUTING.md (hunk 1 above)
Comment (yeskunall) [reply CONTRIBUTING.md]: I’ll answer here as well: the `.env` file isn’t committed. Take a closer look. We’re copying the contents of `.env.example` into `.env` and `.env` is in our `.gitignore`.

**Pull request merged**